en/docs/reference/connectors/google-ads-connector/google-ads-configuration.md

```diff
@@ -0,0 +1,725 @@
+# Google Ads Connector Reference
+
+The following operations allow you to work with the Google Ads API. Click an operation name to see parameter details and samples on how to use it.
+
+## Initialize the connector
+
+To use the Google Ads connector, first create the connection with your configuration. When calling a Google Ads operation, ensure the connection is referenced using the `configKey` attribute.
+
+??? note "init"
+    The `init` operation initializes the connection to Google Ads API.
+    <table>
+        <tr>
+            <th>Parameter name</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+        <tr>
+            <td><code>name</code></td>
+            <td>Unique name to identify the connection.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>base</code></td>
+            <td>The service root URL. The default value is `https://googleads.googleapis.com`.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>clientId</code></td>
+            <td>Client ID of the registered application.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>clientSecret</code></td>
+            <td>Client secret of the registered application.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>refreshToken</code></td>
+            <td>The refresh token that can be used to obtain a new access token.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>developerToken</code></td>
+            <td>The developer token for the Google Ads connection.</td>
+            <td>Yes</td>
+        </tr>
+        <tr>
+            <td><code>loginCustomerId</code></td>
+            <td>The ID of the manager account. Required when using a manager account to access a customer account.</td>
+            <td>No</td>
+        </tr>
+        <tr>
+            <td><code>tokenEndpoint</code></td>
+            <td>The HTTP endpoint that can be uses to obtain an access token. The default value is `https://www.googleapis.com/oauth2/v3/token`.</td>
+            <td>Yes</td>
+        </tr>
+    </table>
+
+    **Sample configuration**
+
+    ```xml
+    <googleAds.init>
+        <connectionType>googleAds</connectionType>
+        <name>GGOGLE_ADS_CONN</name>
```

Comment:
Is `GGOGLE` a typo in the documentation? There are multiple occurrences.

Reply:
Fixed with [70b7f97](https://github.com/wso2/docs-mi/pull/933/commits/70b7f970e86199fba8829404016b58ac512ab3a9)